Comp2024/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -203,8 +206,9 @@ private void configureButtonBindings( )
     // Operator Left/Right Trigger
     // Xbox enums { leftX = 0, leftY = 1, leftTrigger = 2, rightTrigger = 3, rightX = 4, rightY = 5}
     // Xbox on MacOS { leftX = 0, leftY = 1, rightX = 2, rightY = 3, leftTrigger = 5, rightTrigger = 4}
-    m_operatorPad.leftTrigger(Constants.kTriggerThreshold).onTrue(new Dummy("oper left trigger"));
-    m_operatorPad.rightTrigger(Constants.kTriggerThreshold).onTrue(new Dummy("oper right trigger"));
+    m_operatorPad.rightTrigger(Constants.kTriggerThreshold).onTrue(new IntakeRollerRun(m_intake, INRollerMode.ROLLER_EXPEL));
+    m_operatorPad.rightTrigger(Constants.kTriggerThreshold).onFalse(new IntakeRollerRun(m_intake, INRollerMode.ROLLER_STOP));
+    m_operatorPad.leftTrigger(Constants.kTriggerThreshold).onTrue(new Dummy("oper right trigger"));
```

Comment:
The same with this where I think this should be: Dummy("oper left trigger"), instead of "right."